opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java

```diff
@@ -126,8 +146,6 @@ protected void onSpanFinished(MockSpan mockSpan) {
         };
 
         Propagator BINARY = new Propagator() {
-            static final int BUFFER_SIZE = 128;
```

Comment:
This change was not here before. We can clean up (potential) dead code in another PR, to keep changes uniform ;)

Reply:
Sure thing, I have reverted the commit about this.